yt/visualization/plot_modifications.py

```diff
@@ -2471,10 +2470,9 @@ def __call__(self, plot):
         if self.time:
             # If no time_units are set, then identify a best fit time unit
             if self.time_unit is None:
-                if plot.ds.unit_system.name.startswith("us"):
-                    # if the unit system name startswith "us", that means it is
-                    # in code units and we should not convert to seconds for
-                    # the plot.
+                if plot.ds.unit_system._code_flag:
+                    # if the unit system is in code units
```

Comment:
maybe add another comment that this attribute is added from the Dataset class. It feels like I would expect it to be a normal attribute (defined in unyt) if I wasn't specifically reviewing this bit.